clan_members_rank.py

```diff
@@ -71,13 +71,14 @@ def get_clan_cards_rank(self, clan_tag):
             member_card_levels = {
                 'tag': member['tag'],
                 'name': member['name'],
-                'card_level_counts': self.get_card_level_counts(member['cards'])
+                'card_level_counts': self.get_card_level_counts(member['cards'], card_type_filter)
             }
             member_cards_ranked.append(member_card_levels)
             # debugging
             # print(member_card_levels)
 
         # Rank members then return results
+        # TODO move the sorting to a function so that we can unit test just the storting on it's own lol

```

Comment:
So....are you going to do the TODO in this PR?

Reply:
done